scripts/release/build/image_build_process.py

```diff
@@ -167,6 +195,10 @@ def check_if_image_exists(self, image_tag: str) -> bool:
             f"PodmanImageBuilder does not support checking if image exists remotely. Skipping check for {image_tag}.")
         return False
 
+    def get_manfiest_list_digest(self, image) -> Optional[str]:
+        logger.warning(f"PodmanImageBuilder does not support getting digest for manifest list, returning empty digest.")
```

Comment:
nit: I believe we won't run Podman builder to create `release-info.json` so can as well just raise Exception here